contributor-guide/modules/ROOT/pages/release-notes.adoc

```diff
@@ -1,5 +1,129 @@
+
+== Consistency and Formatting
+
+* Use clear and concise language. Experience has shown that short release notes are read, long files much less so. Do not add into release notes extensive text that should be in your main library documentation.
+
+* Be consistent in the formatting. If you're using bullet points for one section, maintain that for others.
+* Use headers and sub-headers to clearly differentiate sections.
+* Provide links to:
+[circle]
+    ** Your main library documentation, if there is updated text, examples or tutorials
+    ** Relevant issues, discussions or threads
+
+== Examples
+
+The following examples show some different approaches. Note how many rely on links to issues to provide the detailed information.
+
+* https://www.boost.org/doc/libs/1_83_0/libs/json/doc/html/json/release_notes.html[JSON Release Notes]
+* https://www.boost.org/doc/libs/1_83_0/libs/nowide/doc/html/changelog_page.html[NoWide Change Log]
+* https://www.boost.org/doc/libs/1_83_0/libs/type_traits/doc/html/boost_typetraits/history.html[Type Traits History]
+* https://www.boost.org/doc/libs/1_81_0/libs/variant2/doc/html/variant2.html#changelog[Variant2 Revision History]
+
+== Update Boost History
+
+When you have completed the library release notes, add the required information to the `[section New Libraries]` or `[section Updated Libraries]` of the https://github.com/boostorg/website/tree/master/feed/history[boostorg/website/feed/history]. Copy the formatting of the examples below.
+
+The examples below come from the https://github.com/boostorg/website/blob/master/feed/history/boost_1_83_0.qbk[boost_183_0.qbk] file:
+
+[source,bash]
+----
+[section New Libraries]
+
+[/ Example:
+* [phrase library..[@/libs/accumulators/ Accumulators]:] Framework for
+  incremental calculation, and collection of statistical accumulators, from Eric Niebler.
+]
+
+* [phrase library..[@/libs/compat/ Compat]:] A repository of C++11 implementations of
+  standard components added in later C++ standards, from Peter Dimov and contributors.
+
+[endsect]
+
+[section Updated Libraries]
+
+[/ Example:
+* [phrase library..[@/libs/interprocess/ Interprocess]:]
+  * Added anonymous shared memory for UNIX systems.
+  * Move semantics for shared objects ([ticket 1932]).
+  * Conform to `std::pointer_traits` requirements ([github_pr interprocess 32])
+  * Fixed `named_condition_any` fails to notify ([github interprocess 62])
+]
+
+* [phrase library..[@/libs/any/ Any]:]
+  * New `boost::anys::unique_any` C++11 class - an alternative to
+    `boost::any` (or to `std::any`) that does not require copy
+    or move construction from the held type.
+  * Ported documentation to Quickbook, Doxygen comments now used in code.
+
+  ....
+
+  [endsect]
+----
+
+== See Also
+
+* https://github.com/boostorg/website/tree/master/feed/history[boostorg/website/feed/history]
```

Comment:
When this is rendered, it looks weird. Should we give this a proper name?

Reply:
Agreed, changed the name to Boostorg History